src/System.Buffers.ReaderWriter/System/Buffers/Writer/OutputWriter.cs

```diff
@@ -6,31 +6,23 @@
 
 namespace System.Buffers.Writer
 {
-    public static class OutputWriter
-    {
-        public static OutputWriter<T> Create<T>(T output) where T : IBufferWriter<byte>
-        {
-            return new OutputWriter<T>(output);
-        }
-    }
-
-    public ref struct OutputWriter<T> where T : IBufferWriter<byte>
+    public ref partial struct BufferWriter<T> where T : IBufferWriter<byte>
```

Comment:
Since now it's `BufferWriter<T>`, the file should be renamed.

Reply:
Yeah, I will rename it in a subsequent change, otherwise github gets often confused about the change and treats files are deleted/added.